### ISSUE TITLE
1.16.0 > Bug > Card > Titles always render, even when empty

### ISSUE THREAD

**Issue by oodavid** [react-native-training/react-native-elements#566]
### Description

I updated to **1.16.0** and now all `<Card>` elements render with a title component, even if the title is empty.

### Reproduction Steps and Sample Code

```
<Card
  image={{ uri: 'https://pbs.twimg.com/profile_banners/18031588/1456170364/1500x500' }}>
  <Text>Hello World</Text>
</Card>
```

```
<Card>
  <Text>Hello World</Text>
</Card>
```

https://snack.expo.io/SkNRwitOb

### Solution

I fixed by downgrading back to **1.15.0**

**Comment by iRoachie** [comment]
Thanks will work on this ASAP

**Comment by iRoachie** [comment]
@oodavid It's fixed on `next`. The fix will be available in the next release 0.17. For the meantime, you can use the `next` branch.

**Comment by iRoachie** [comment]
@mobiusnz Glad you got it figured out. Will try to publish this weekend
